### DISCUSSION TITLE
Framework de CSS

### DISCUSSION THREAD

**Discussion by Israel-12-u** [Launch-X-Latam/MisionFrontEnd#34] [Q&A]
se ocupara algún framework de CSS durante este periodo, como Bulma o Bootstrap, o será desarrollado desde cero ?

**Comment by doguedogue** [comment]
La idea aqui es conocer las bases de cero. 🚀😀

Posteriormente es bueno usar algún framework como dices, incluso metodologías (BEM por ejemplo) y hasta el uso de CSS como lenguaje de programación (KHE KHEEEE) mmju como lenguaje.

**Comment by Israel-12-u** [comment]
Hola, veo que todos están haciendo equipo para el Hackathon, si alguien gusta estoy libre, conocimiento en Lenguajes como Python,Java y C++, bases de datos como MYSQL,Postgrest,SQLite3, MongoDB, además de usar framewors de desarrollo web. al igual que para el desarrollo de APIS.
Dejo mi Correo de innovación por si alguien gusta agregarme a su equipo: lx04770@innovaccion.mx
